apps/api/tests/test_admin.py

```diff
@@ -119,7 +120,7 @@ def test_can_include_decision_from_reviews() -> None:
         },
     }
 
-    admin._include_review_decision(record)
+    applicant_review_processor.include_review_decision(record)
```

Comment:
Chore: since this test (and the one below) are now about functions in the new module, it would also make more sense to move it to the new test file.